var/spack/repos/builtin/packages/swan/package.py

```diff
@@ -0,0 +1,37 @@
+from spack import *
```

Comment:
Please restore the standard Spack package copyright.

Reply:
fixed